### PR TITLE
Added alert for the firewall lost connection

### PR DESCRIPTION
**Pull request by hardikhdholariya**

### CODE REVIEW

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 1]
```diff
@@ -471,6 +471,41 @@ action.cyences_notable_event_action.products = Sophos Endpoint Protection
...
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 2,17,32,47 * * * *
+description = This alert will trigger when a Firewall lost connection to Sophos Central. \
+\
+Data Collection - Sophos Central Add-on for Splunk (https://splunkbase.splunk.com/app/6186/)
+dispatch.earliest_time = -17m@m
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:487]: It is that important to be alerting every 15 minutes or can we do hourly??

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 2]
```diff
@@ -471,6 +471,41 @@ action.cyences_notable_event_action.products = Sophos Endpoint Protection
...
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search =`cs_sophos` sourcetype="sophos_events" type="Event::Firewall::LostConnectionToSophosCentral" \
+| stats count, latest(_time) as _time, values(name) as threat, values(source_info.ip) as src_ip by host, location | sort -count \
+| eval cyences_severity = "critical" \
+| `cs_human_readable_time_format(_time, event_time)` \
+| `cs_sophos_firewall_lost_connection_to_sophos` 
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_sophos_firewall_lost_connection_to_sophos
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:500]: Have we added the macro in macros.conf?

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 1 above)
Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/savedsearches.conf:487]: I think we can do it hourly to reduce the alerting as sometimes it has 1-2 events hourly for same host and location.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 3]
```diff
@@ -471,6 +471,41 @@ action.cyences_notable_event_action.products = Sophos Endpoint Protection
...
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_sophos` sourcetype="sophos_events" type="Event::Firewall::LostConnectionToSophosCentral" \
+| stats count, latest(_time) as _time, values(name) as threat, values(source_info.ip) as src_ip by host, location | sort -count \
+| eval cyences_severity = "critical" \
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:496]: Have we verified the severity with the requester??

**Review by VatsalJagani — APPROVED** [review]

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 3 above)
Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/savedsearches.conf:496]: verified and updated it to high.

**Pull request merged**